src/main/java/org/spideruci/tacoco/analysis/AbstractAnalyzer.java

```diff
@@ -76,6 +83,20 @@ public static AbstractAnalyzer getInstance() {
 		}
 	}
 	
+	public HashSet<String> getIgnoreTestClassSet()
```

Comment:
What is the philosophy behind having `getIgnoreTestClassSet` be a part of the `AbstractAnalyzer`? Particularly, why place this method in any of the `Analyzer` classes? It seems particularly strange to place this method in the `AbstractAnalyzer` given that tests/ignored-tests are only make sense when you are talking about `AbstractRuntimeAnalyzer`. 

Here's are two ways to go about this: 
Let's come up with a good reason, as to why we should place this code in any of the `Analyzer` classes. And put that reason in the Javadocs of this method. AND IF are going to place it in the `Analyzer` classes let's consider putting it in the `AbstractRuntimeAnalyzer` class.
OR
If we do not have a good reason to place it in the `Analyzer` classes, then let's find a better place to put this, like the `AnalyzerCli` class, where you can make this a static method, with some name like `readIgnoredTestClasses`.